js/product-details.js

```diff
@@ -0,0 +1,255 @@
+    valueConteinerBig.classList.add("product-details-value--big--" + key);
+    for (i = 0; i < drink[key].length; i++) {
+      let textSmoll = document.createElement("div");
+      textSmoll.innerHTML = drink[key][i];
+      textSmoll.classList.add("details-notes--margin");
+      valueConteinerSmoll.appendChild(textSmoll);
+      let textBig = document.createElement("div");
+      textBig.innerHTML = drink[key][i];
+      textBig.classList.add("details-notes--margin");
+      valueConteinerBig.appendChild(textBig);
+    }
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    conteinerBig.appendChild(valueConteinerBig);
+  } else {
+    let valueConteinerSmoll = document.createElement("div");
+    valueConteinerSmoll.innerHTML = drink[key];
+    valueConteinerSmoll.classList.add("product-details-value--smoll");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--none");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--" + key);
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    let valueConteinerBig = document.createElement("div");
+    valueConteinerBig.innerHTML = drink[key];
+    valueConteinerBig.classList.add("product-details-value--big");
+    valueConteinerBig.classList.add("product-details-value--big--none");
+    valueConteinerBig.classList.add("product-details-value--big--" + key);
+    conteinerBig.appendChild(valueConteinerBig);
+  }
+};
+
+const clickButton = function() {
+  const productDetails = document.getElementsByClassName("details-button");
+  Array.from(productDetails).forEach(function(item) {
+    item.addEventListener("click", function() {
+      if (windowWidth() > 769) {
+        remuveClass();
+        this.classList.add("details-button-active--big");
+        let valueConteiner = document
+          .getElementsByClassName(
+            "product-details-value--big--" + this.innerHTML
+          )[0]
+          .classList.toggle("product-details-value--big--none");
+      } else {
+        this.classList.toggle("details-button-active--smoll");
+        document
+          .getElementsByClassName(
+            "product-details-value--smoll--" + this.innerHTML
+          )[0]
+          .classList.toggle("product-details-value--smoll--none");
+      }
+    });
+  });
+};
+
+const remuveClass = function() {
+  const productDetails = document.getElementsByClassName("product-details")[0]
+    .children;
+  for (i = 0; i < productDetails.length; i++) {
+    productDetails[i].classList.remove("details-button-active--big");
+  }
+  const productValueBig = document.getElementsByClassName(
+    "product-details-value"
+  )[0].children;
+  for (i = 0; i < productValueBig.length; i++) {
+    productValueBig[i].classList.add("product-details-value--big--none");
+  }
+};
+
+const windowWidth = function() {
+  const width = document.getElementsByClassName("content-width")[0].offsetWidth;
+  return width;
+};
+
+// fix fo IE (now he know what is from)
+if (!Array.from) {
```

Comment:
I found where did you pase code from.
are you ready to explain me each string of this code?
if Array from is not supported in IE - just find another way.
google how to make array from DOM nodes, please. it's 9 sybmols instead of all of this shit :)
remove